charts/keycloak/values.yaml

```diff
@@ -48,10 +28,28 @@ keycloak:
         cpu: "500m"
         memory: "512Mi"
 
+ingress:
+  enabled: true
+  domain: keycloak.ci.fairway.app
```

Comment:
I don't think we need a default value here. Maybe good to have a matching PR already in the https://github.com/thehyve/fairspace-config-private repo?

Reply:
Done, PR: https://github.com/thehyve/fairspace-config-private/pull/1